src/main.c

```diff
@@ -487,7 +490,9 @@ static error_t parse_opt(int key, char *arg, struct argp_state *state) {
 	    strcpy(synclogfile, arg);
 	    break;
 	case 'd':		// debug rigctl
-	    debugflag = true;
+	    debuglevel = atoi(arg);
```

Comment:
could a numeric argument be checked? `tlf -dx` starts just fine

Reply:
see commit bc4dda82